### DISCUSSION TITLE
Coastal Coupling meeting May 2023

### DISCUSSION THREAD

**Discussion by saeed-moghimi-noaa** [oceanmodeling/Management#14] [General]
Please make a note here to get included in the **oceanmodeling** organization.

**Comment by sadaf-mahmoudi96** [comment]
Hello,
My name is Sadaf Mahmoudi, a Ph.D. student at the University of Alabama. My research is mainly focused on coastal flooding, sea level rise, and flooding thresholds. I would like to be added to this organization as it brings beneficial opportunities for my research.

**Comment by saeed-moghimi-noaa** [comment]
Hi @sadaf-mahmoudi96 

I just sent you an invite to become a member.  We also meet every 2 weeks to discuss developing various tools including https://github.com/oceanmodeling/searvey .

Let me know if you want to be invited. You do not need to attend always it will be an optional invitation when you have time. However it is a good way to start interacting with NOAA developers.

Thanks for your interest,
@saeed-moghimi-noaa

**Comment by sadaf-mahmoudi96** [comment]
Hello,

Are the meetings being held virtually?

Regards,
Sadaf

Sadaf Mahmoudi Kouhi, Ph.D. Student
Coastal Hydrology Lab
Department of Civil, Construction, and Environmental Engineering
The University of Alabama
Office: ​Cyber Hall, Turing Wing
Email: ***@***.***
________________________________
From: Saeed Moghimi ***@***.***>
Sent: Wednesday, May 24, 2023 2:32 PM
To: oceanmodeling/Management ***@***.***>
Cc: Sadaf Mahmoudikouhi ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [oceanmodeling/Management] Coastal Coupling meeting May 2023 (Discussion #14)

You don't often get email from ***@***.*** Learn why this is important<https://aka.ms/LearnAboutSenderIdentification>

@sadaf-mahmoudi96<https://github.com/sadaf-mahmoudi96>

I just sent you an invite to become member. We meet every 2 weeks to discuss developing various tools including https://github.com/oceanmodeling/searvey .

Let me know if you want to be invited. You do not need to attend always it will be an optional invitation however it is a good way to start interacting with NOAA developers.

Thanks for you interest,
@saeed-moghimi-noaa<https://github.com/saeed-moghimi-noaa>

**Comment by saeed-moghimi-noaa** [comment]
Yes, on google meet.  Please feel free to call in here:

Observational Data (and other modeling tools) Collaboration
Wednesday, June 7 · 10:00 – 10:40
Time zone: America/New_York
Google Meet joining info
Video call link: https://meet.google.com/rvr-dnoh-iwy
Or dial: ‪(US) +1 319-382-9678‬ PIN: ‪388 031 212‬#
More phone numbers: https://tel.meet/rvr-dnoh-iwy?pin=8152807019250

You can send me your email so I can include you in the calendar.



______________________________________________________
Saeed Moghimi, PhD
*NOS Storm Surge Modeling
<https://nauticalcharts.noaa.gov/learn/storm-surge-modeling.html> Team Lead*
Coastal Marine Modeling Branch, Coast Survey Development Laboratory, Office
of Coast Survey at NOAA National Ocean Service.
Address: 1315 East West Hwy, Room 6607, Silver Spring, Maryland  20910

Phone: ‪(302) 827-3986

The contents of this message are mine personally and do not necessarily
reflect any position of NOAA.

**Comment by sadaf-mahmoudi96** [comment]
Sure, my email is ***@***.***
Thank you!

Sadaf Mahmoudi Kouhi, Ph.D. Student
Coastal Hydrology Lab
Department of Civil, Construction, and Environmental Engineering
The University of Alabama
Office: ​Cyber Hall, Turing Wing
Email: ***@***.***

**Comment by saeed-moghimi-noaa** [comment]
Your email was stripped out by gtihub. Please just go ahead and call in to
the meeting and we will get your email included.
______________________________________________________
Saeed Moghimi, PhD
*NOS Storm Surge Modeling
<https://nauticalcharts.noaa.gov/learn/storm-surge-modeling.html> Team Lead*
Coastal Marine Modeling Branch, Coast Survey Development Laboratory, Office
of Coast Survey at NOAA National Ocean Service.
Address: 1315 East West Hwy, Room 6607, Silver Spring, Maryland  20910

Phone: ‪(302) 827-3986

The contents of this message are mine personally and do not necessarily
reflect any position of NOAA.

**Comment by sadaf-mahmoudi96** [comment]
Thank you!

Sadaf Mahmoudi Kouhi, Ph.D. Student
Coastal Hydrology Lab
Department of Civil, Construction, and Environmental Engineering
The University of Alabama
Office: ​Cyber Hall, Turing Wing
Email: ***@***.***

**Comment by dpsnowden** [comment]
HI @saeed-moghimi-noaa Please add me to the discussion.  Thx!

**Reply by saeed-moghimi-noaa** [reply]
Please check your inbox

**Comment by SODaramola** [comment]
Hi Dr Moghimi, I would love to be a member of the community. Please add me.

**Reply by saeed-moghimi-noaa** [reply]
Check your inbox

**Reply by SODaramola** [reply]
Thank you, Dr Moghimi

**Comment by ebrahimhamidi** [comment]
Hello,
I would like to be a part of this code community.
Thanks for your excellent job,

**Reply by saeed-moghimi-noaa** [reply]
See your inbox.

**Reply by ebrahimhamidi** [reply]
Thank you, Saeed, that is all set.

**Comment by aminilia** [comment]
Hi Saeed, Could you please add me to this community code? Thanks, Amin

**Reply by saeed-moghimi-noaa** [reply]
Done. Check your inbox.

**Comment by kiaghadi2013** [comment]
HI @saeed-moghimi-noaa Please add me to the discussion. Thx!
My email is amin.kiaghadi at twdb.texas.gov

**Reply by saeed-moghimi-noaa** [reply]
Please accept the invitation.

**Comment by tracyfanara** [comment]
@sadaf-mahmoudi96 please add me

**Reply by saeed-moghimi-noaa** [reply]
Hi @tracyfanara , Just sent invitation. Please accept.
